t/ui/16-activity-view.t

```diff
@@ -94,3 +94,17 @@ subtest 'Current jobs' => sub {
 
```

Comment:
Can we cover the "new" states in the test? Maybe by adding to `%fake_events`?

Reply:
Yes, I'm working on that. I also need to create new jobs for that with all combinations.